.travis.yml

```diff
@@ -27,3 +27,4 @@ env:
     # prevent wine popup dialogs about installing additional packages
     - WINEDLLOVERRIDES="mscoree,mshtml="
     - WINEDEBUG="-all"
+    - DEBUG=electron-packager
```

Comment:
Are you okay with the other changes in this pull request, the added `debug` calls? If so, I'll merge this if the build passes.